modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolatorSegregated.C

```diff
@@ -155,8 +155,8 @@ VectorValue<ADReal>
 INSFVRhieChowInterpolatorSegregated::getVelocity(const Moose::FV::InterpMethod m,
                                                  const FaceInfo & fi,
                                                  const Moose::StateArg & /*time*/,
-                                                 const THREAD_ID /*tid*/
-) const
+                                                 const THREAD_ID /*tid*/,
+                                                 const bool /*subtract_mesh_velocity*/) const
```

Comment:
should you assert on subtract_mesh_velocity since it s not implemented

Reply:
I think I will just error if this object is `_displaced`. It's valid for a "general" postprocessor to call this method with `true` as long as the calculation is not actually displaced